RealmApp.tsx

```diff
@@ -1,9 +1,9 @@
-import Realm from "realm";
+import Realm from 'realm';
 
-let mongoId = process.env.REACT_APP_ID;
+let mongoId = 'process.env.REACT_APP_ID';
 if (!mongoId) {
-  console.error("No MongoDB App ID");
-  mongoId = mongoId = "";
+  console.error('No MongoDB App ID');
+  mongoId = mongoId = '';
```

Comment:
Seems like this mongoId = mongoId is a remnant of a separate merge.. we should probably remove this